anyway/utilities.py

```diff
@@ -195,3 +198,36 @@ def chunks(l, n):
     """Yield successive n-sized chunks from l."""
     for i in range(0, len(l), n):
         yield l[i : i + n]
+
+
+def is_valid_number(phone: str) -> bool:
+    try:
+        phone_obj = phonenumbers.parse(phone, "IL")
+        return phonenumbers.is_valid_number(phone_obj)
+    except NumberParseException:
+        return False
+
+
+def check_is_a_safe_redirect_url(url: str) -> bool:
```

Comment:
The prefix `check_` is inconsistent with the previous function, makes the name longer without adding any information, and results in less "nice" client code (`if is_safe(...):` is nicer than `if check_is_safe(...)`).